posthog/models/person/person.py

```diff
@@ -99,3 +99,24 @@ class Meta:
 
     # current version of the id, used to sync with ClickHouse and collapse rows correctly for new clickhouse table
     version: models.BigIntegerField = models.BigIntegerField(null=True, blank=True)
+
+
+class PersonOverride(models.Model):
```

Comment:
I'm guessing there will be a similar table in CH for querying? and both of these stay in sync?

Reply:
yes - works very similarly to Person DistinctID. 

Why we need the table in postgres is explained here: https://docs.google.com/document/d/12Q1KcJ41TicIwySCfNJV5ZPKXWVtxT7pzpB3r9ivz_0/edit#bookmark=id.gpwgfub6blq9

Actually a bit more complex compared to distinct ID as we'll have 2 separate tables in CH (short and long term but exactly the same schema, we'll union them and then join for queries), read more: https://docs.google.com/document/d/12Q1KcJ41TicIwySCfNJV5ZPKXWVtxT7pzpB3r9ivz_0/edit#bookmark=id.9gse52dl0e4w